install.md

```diff
@@ -1,4 +1,4 @@
-# Ubuntu installation instructions
+# Installation instructions.
```

Comment:
Yeah I totally agree, however I am thinking of something like. 

### In base OS

### In Docker

Since Docker will be using ubuntu base image in this case.